ansible/roles/postgresql/tasks/main.yml

```diff
@@ -92,17 +92,52 @@
   when:
     pg_install == True
 
+- name: Create PostgreSQL ICDS users
```

Comment:
why did you split this up again? seemed fine to create all the users with this task